src/kudu/tserver/simple_tablet_manager.cc

```diff
@@ -438,6 +438,14 @@ Status TSTabletManager::SetupRaft() {
 
   InitLocalRaftPeerPB();
 
+  // If the persistent vars file does not already exist, create one
+  if (!persistent_vars_manager_->PersistentVarsFileExists(kSysCatalogTabletId)) {
```

Comment:
does this call enable us to kill the same call in CreateNew flow?

Reply:
Done